# Raw Velodyne LIDAR packet.

time stamp             
uint8[1206] data 

